source/org/intellij/plugins/ceylon/ide/project/CeylonPageTwo.ceylon

```diff
@@ -300,10 +302,16 @@ shared class CeylonPageTwo()
     shared actual void load(IdeaCeylonProject config) {
         form.\imodule = config.ideArtifact;
         value systemRepository = config.ideConfiguration.systemRepository;
-        form.systemRepository.text = if (!exists systemRepository) then null else systemRepository.string;
```

Comment:
Sure! Why not. I tried to make minimum changes to make it compatible with IDEA 2017. But if you want I can make some code style improvements later in separate commit.